algorithms/unit_tests/TestRandom.hpp

```diff
@@ -486,6 +775,13 @@ void test_random_xorshift64() {
   Impl::TestDynRankView<ExecutionSpace,
                         Kokkos::Random_XorShift64_Pool<ExecutionSpace>>(10000)
       .run();
+#if defined(KOKKOS_ENABLE_SYCL) || defined(KOKKOS_ENABLE_CUDA) || \
+    defined(KOKKOS_ENABLE_HIP)
+  const int N = 134217728;
+#else  // SERIAL, HPX, OPENMP
+  const int N         = 134217728 / 8;
+#endif
```

Comment:
How long does this typically take?

Reply:
@masterleinad With these N, the random tests takes 47s on cuda backend, 8s on openmp backend, and 160s on serial backend. I ran the tests on Lassen with V100 GPU and POWER9 CPU.

Reply:
@dalg24 Isn't that 160 seconds on SERIAL backend big enough? I have no problem with just using ```N = 134217728``` for all backends. I am going to change this.